public/index.html

```diff
@@ -47,7 +47,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>hetmech · hetnet connectivity search</title>
+    <title>hetnet connectivity search</title>
```

Comment:
I don't have a capitalization preference. Will leave aesthetic details up to @vincerubinetti

Reply:
I don't think this matters because we're going to change it later anyway. This app is eventually going to be a sub-element of an existing page on het.io. We can decide on what the page titles should all be for het.io when we do that redesign.